### PR TITLE
OM-3044: Integrate APIs in Company Instance Creation form

### PR DESCRIPTION
**Pull request by amar-hunar**
This PR addresses part of [OM-3044](https://hunar.atlassian.net/browse/OM-3044)

## Features:

- add APIs for company creation and integrate them into `<CompanyCreationForm />`
- add company creation status

## Technical:
- The following components are added:
  - `DomainUpdateStatus`
- The following hooks are added:
  - `useAddDNSRecord`
  - `useAddDomainAlias`
  - `useCreateCompany`
- The following packages are installed:
  - `@chakra-ui/icons`
- The following assets are added:
  - `google.png`
  - `netlify.png`
  - `retry.svg`

### CODE REVIEW

File: src/containers/CompanyContainer.tsx [hunk 1]
```diff
@@ -1,11 +1,169 @@
...
+    }, [
+        addDNSRecord.isLoading,
+        addDomainAlias.isLoading,
+        createCompany.isLoading
+    ]);
+
+    const createDNSRecord = (companyId: string) => {
+        // TODO: Remove two lines below once testing is done
+        setShowDNSStatus(true);
+        return;
```
Comment (amar-hunar) [inline src/containers/CompanyContainer.tsx:52]: NOTE: these two lines are added for QA testing and will removed once QA is complete

File: src/containers/CompanyContainer.tsx [hunk 2]
```diff
@@ -1,11 +1,169 @@
...
+                    setDnsRetryCount(count => count + 1);
+                }
+            }
+        );
+    };
+
+    const createDomainAlias = (companyId: string) => {
+        // TODO: Remove two lines below once testing is done
+        setShowDomainAliasStatus(true);
+        return;
```
Comment (amar-hunar) [inline src/containers/CompanyContainer.tsx:71]: NOTE: these two lines are added for QA testing and will removed once QA is complete

File: src/components/company/DomainStatus.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    iconSrc: string;
+    title: string;
+    isSuccessful: boolean;
+    isRetrying: boolean;
+    isRetryVisible: boolean;
+    errorMessage?: string;
+    onRetryClick: VoidFunction;
+}
+
+export const DomainStatus = ({
```
Comment (misss-popcorn) [inline src/components/company/DomainStatus.tsx:17]:
```suggestion
export const CompanyDomainStatus = ({
```

File: src/containers/CompanyContainer.tsx [hunk 4]
```diff
@@ -1,19 +1,166 @@
...
+    };
+
+    const onSubmitClick = (form: CompanyFormProps) => {
+        createCompany.mutate(
+            {
+                params: { companyId: form.companyId },
+                requestBody: form
+            },
+            {
+                onSuccess: () => {
```
Comment (misss-popcorn) [inline src/containers/CompanyContainer.tsx:90]: why are we doing all of this here? and not in companyAddForm?

Comment (amar-hunar) [reply src/containers/CompanyContainer.tsx]: Moved this method to `CompanyAddForm`

**Pull request merged**